js/analyzescorer.js

```diff
@@ -11,7 +11,7 @@ var assessments = {};
 assessments.wordCount = require( "./assessments/countWords.js" );
 assessments.fleschReading = require( "./assessments/calculateFleschReading.js" );
 assessments.linkCount = require( "./assessments/getLinkStatistics.js" );
-assessments.pageTitleKeyword = require( "./assessments/pageTitleKeyword.js" );
+assessments.pageTitleKeyword = require( "./assessments/pagetitleKeyword.js" );
```

Comment:
If we are to follow proper camel case practices, the file should be renamed and not the reference to the file.
`pageTitleKeyword` would be the proper way of naming this file.